src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -146,11 +146,7 @@ public Task<IEnumerable<TResult>> ReadAsync<TResult>(string query, Action<IDbCom
                 throw new ArgumentNullException("selector");
```

Comment:
@jdom That's true for that specific comment (though in the spirit of optimizations there's at least `ValueTask` and maybe `IAsyncEnumeable`. But wait a moment, do you mean even after my purported fixes there are regressions that shouldn't exist? I see there's https://ci.dot.net/job/dotnet_orleans/job/master/job/netfx-functional_prtest/461/testReport/UnitTests.MembershipTests/SqlServerMembershipTableTests/MembershipTable_SqlServer_UpdateRowInParallel/ for instance and it's flagged as regression. I can't reproduce that, but it could be due to performance problems, though primary key violation could easily indicate something else too (though in that case I suppose there would be a retry).